packages/drawer/src/Default/Default.types.ts

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 import * as bs from 'react-bootstrap';
 
+// eslint-disable-next-line import/namespace
+export interface DrawerHeader {
+  content: React.ReactNode;
+  bsProps: bs.OffcanvasHeaderProps;
+}
+
+export interface DrawerProps {
+  header: DrawerHeader;
+  body: React.ReactNode;
+}
 export interface DrawerDefaultCommons {
```

Comment:
I've updated the types so we pass an object like with html elements or react components instead of text.
```js
const drawer: DrawerProps = {
  header: {
    content: <h2>Drawer Header</h2>,
    bsProps: {
      closeButton: true,
    },
  },
  body: (
    <p>
      Some text as <em>placeholder</em>. In real life you can have the elements
      you have chosen. Like, text, images, lists, etc.
    </p>
  ),
};
```